components/permit-holders/MedicalHistoryCard.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Text, Divider, Link } from '@chakra-ui/react'; // Chakra UI
+import Table from '@components/internal/Table'; // Table component
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard';
+import { Aid, MedicalInformation } from '@lib/graphql/types'; // Aid enum & MedicalInformation type
+import MedicalHistoryModal from '@components/permit-holders/modals/MedicalHistoryModal'; // Medical History Modal
+
+// Placeholder data
+const mockMedicalHistory = {
+  disability: 'Condition 1',
+  affectsMobility: true,
+  mobilityAidRequired: true,
+  cannotWalk100m: true,
+  aid: [Aid.Cane, Aid.ElectricChair],
+  notes: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  certificationDate: '2021/01/01',
+};
+
+const DATA = Array(4).fill({
+  disablingCondition: 'Condition 1',
+  associatedApp: { associatedApp: 12345 },
+  dateUploaded: '2021/01/01',
+  fileUrl: { fileUrl: '/' },
+  application: { application: mockMedicalHistory },
+});
+
+const COLUMNS = [
+  {
+    Header: 'Disabling Condition(s)',
```

Comment:
(more of a design critique but) I think we should change "Disabling Condition(s)" to "Disabling Condition". Unless multiple conditions will go in the same row, we don't need it, and even in that case it still seems unnecessary to have that "(s)" there.

Reply:
So medical history shows the permit holder's medical conditions at the time of that application. Meaning the first row is all their medical conditions at the time of the most recent application, and the second row is all their medical conditions at the time of the application prior. So multiple conditions can go into one row, but I do agree though that the "(s)" isn't the most aesthetically pleasing. I'll bring it up with the designers and get their input

Reply:
Dropped the "(s)" after getting Andy's input + approval